sopel/modules/currency.py

```diff
@@ -1,102 +1,189 @@
 # coding=utf-8
 # Copyright 2013 Elsie Powell, embolalia.com
+# Copyright 2019 Mikkel Jeppesen
 # Licensed under the Eiffel Forum License 2
 from __future__ import unicode_literals, absolute_import, print_function, division
 
+import time
 import re
+import requests
```

Comment:
Another import-sorting prep change.

```suggestion

import requests
```

Reply:
reordered imports to be alphabetical, native python first, sopel modules after.

Reply:
`requests` is not a standard import. It comes from PyPI. This block should actually be two blocks:

```python
import re
import time

import requests
```

Note the blank line.